src/HDInsight/HDInsight/ManagementCommands/GrantAzureHDInsightHttpServicesAccessCommand.cs

```diff
@@ -56,8 +58,10 @@ public override void ExecuteCmdlet()
                 ResourceGroupName = GetResourceGroupByAccountName(ClusterName);
             }
 
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
If you remove the obsolete attributes below, these should be unnecessary

Reply:
> If you remove the obsolete attributes below, these should be unnecessary

Fixed. Have removed the obsolete attributes.